source/includes/_users.md

```diff
@@ -260,6 +314,8 @@ name | string | The User's name.
 email | string | The User's corporate email.
 phone | string | The User's phone.
 password | string | The User's password.
+is_admin | bolean | The User's is_admin.
```

Comment:
specifies whether the user you are creating is an administrator or not.

Reply:
done